src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/models/FizzModel.java

```diff
@@ -0,0 +1,34 @@
+package com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.impl.models;
+
+public class FizzModel extends Model implements com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.models.Model {
+
+    public final static String FIZZ = "Fizz";
+    private final String modelValue;
+
+    public FizzModel(String modelValue) {
+        this.modelValue = modelValue;
+    }
+
+    public FizzModel() {
+        this.modelValue = FIZZ;
+    }
+
+    @Override
+    public String getValue() {
+        return modelValue;
+    }
+
+    public static class Builder {
```

Comment:
We must make the constructor and builder private, and provide a `public Builder builder(String modelValue) { return new Builder(modelValue); }` - further, we should assert that the parameters are not null, empty or blank.